djangocms_versioning/helpers.py

```diff
@@ -43,3 +44,28 @@ def replace_admin_for_models(models, admin_site=None):
         except KeyError:
             continue
         _replace_admin_for_model(modeladmin, admin_site)
+
+
+def register_versionadmin_proxy(version_proxy, grouper_name, admin_site=None):
+    """
```

Comment:
Include some info in the docstring about what this function actually does as well.